mapquester_frontend/package.json

```diff
@@ -29,7 +27,8 @@
     "eslint-plugin-react": "^7.37.1",
     "eslint-plugin-react-hooks": "^5.0.0",
     "postcss": "^8",
+    "recoil": "^0.7.7",
```

Comment:
"recoil" should not be in devDependencies.

Reply:
Okay, making the edit